tests/test_metadata_serialization.py

```diff
@@ -51,8 +51,11 @@ def wrapper(test_cls: "TestSerialization"):
 class TestSerialization(unittest.TestCase):
 
     valid_keys: DataSet = {
+        # Do we want to add unrecognized fields in the Key class?
```

Comment:
```suggestion
```

this may be a valid question but maybe not useful in test code?

Reply:
I will remove it, but wanted to ask raise that question.
Do we want to allow unrecognized fields in the Key class?

Reply:
There's a desire to be able to store unrecognised fields in a [keyval](https://theupdateframework.github.io/specification/latest/#keyval), see https://github.com/theupdateframework/specification/issues/163 and https://github.com/theupdateframework/go-tuf/pull/133